src/SFA.DAS.EAS.Api/Web.config

```diff
@@ -1,389 +1,389 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Did this file really change? I can't see any clear diffs; looks a bit like a complete delete and add.

Was it in commit by mistake? or is it merge noise?